### ISSUE TITLE
Please add option in config to customize bot status message (Playing: custom text)

### ISSUE THREAD

**Issue by thedovy** [KarlOfDuty/SupportBoi#22]

**Comment by Emil8250** [comment]
Hi 

I might take a look at this, if you're taking pull requests?

**Comment by KarlOfDuty** [comment]
I'd agree with the first half but there are so many different embeds that I don't think the colouring stuff is a very good idea.

**Comment by thedovy** [comment]
> I'd agree with the first half but there are so many different embeds that I don't think the colouring stuff is a very good idea.

I'm looking at the code myself, is it possible to set hex color instead of just "Red" or "Green" or "Cyan?
Talking about this part: `Color = DiscordColor.Red,`

**Comment by KarlOfDuty** [comment]
Think so

**Comment by thedovy** [comment]
Maybe you could suggest how should I do that? I tried few different methods and none of them seem to work. But then again, I am very bad at coding:D

**Comment by KarlOfDuty** [comment]
No, I have always just used the existing colours.

**Comment by thedovy** [comment]
> No, I have always just used the existing colours.

I actually got a great idea. In this bot only three colors are being utilized, that is Green, Red, and Cyan.
You could make those 3 colors customizable in the config, for example:
```
colors:
    positive_messages: #00FF00 (Green)
    negative_messages: #FF0000 (Red)
    neutral_messages: #00FFFF (Cyan)
```
I think that would be great solution!



--



--

**Comment by Emil8250** [comment]
I've added 3 colors in the config, like this, on my local version: 

`    # The color of embedded warning messages.
    color-warning: "#FF0000"
    # The color of embedded success messages.
    color-success: "#00FF00"
    # The color of embedded info messages.
    color-info: "#00FFFF"`

Seems to work fine.

**Comment by thedovy** [comment]
> I've added 3 colors in the config, like this, on my local version:
> 
> ```
> # The color of embedded warning messages. 
> color-warning: "#FF0000"
> # The color of embedded success messages.
> color-success: "#00FF00"
> # The color of embedded info messages.
> color-info: "#00FFFF"
> ```
> 
> Seems to work fine.

Is there any way for me to use your version?

**Comment by Emil8250** [comment]
I'm unsure - I've also added a "presence-game" config, which allows you to set which game the bot is currently playing. 

I'm uncertain if Karl is taking pull requests, and if he wants any of this merged in at all :) 

I can take out the coloring part, and merge in the presence part, if that's preferable?

**Comment by thedovy** [comment]
> Hi
> 
> I might take a look at this, if you're taking pull requests?

Well, on this message Karl reacted with thumbs up, so I think you could just go for it

**Comment by thedovy** [comment]
Why did colors were left out:(

**Comment by Emil8250** [comment]
> 
> 
> I'd agree with the first half but there are so many different embeds that I don't think the colouring stuff is a very good idea.

Because of this reply --> I have the feature locally though, so it's easy implemented :)

**Comment by thedovy** [comment]
@KarlOfDuty Please, give this good man permission to add pull request for customizing colors. It's great to have options on stuff like this because every server setup is different :)

**Comment by KarlOfDuty** [comment]
Anyone can make any pull requests they want.

**Comment by KarlOfDuty** [comment]
Closed by #23
